src/naksu/mebroutines/mebroutines.go

```diff
@@ -57,7 +61,7 @@ func Run(commandArgs []string) error {
 
 	err := cmd.Run()
 	if err != nil {
-		ShowWarningMessage(fmt.Sprintf(xlate.Get("command failed: %s"), strings.Join(commandArgs, " ")))
+		ShowWarningMessage(fmt.Sprintf(xlate.Get("command failed: %s (%s)"), strings.Join(commandArgs, " "), getErrorStr(err)))
```

Comment:
Voisiko yksinkertaistaa niin, että format string olisi suoraan "command failed %s (%v)" ja `Sprintf`:n viimeinen parametri olisi `err`? Tällöin voisi poistaa `getErrorStr`-apufunktion.

Reply:
Sama pätee siis kaikkiin muutettuihin `ShowWarningMessage` ja `log.Debug`-kutsuihin tässä commitissa.

Reply:
Mulla oli ajatus, että siellä `getErrorStr`:ssä olisi tehty jotain muutakin formatteerausta, mutta ei sille tosiaan ollut sitten tarvetta. Muutin kuten ehdotit.